internal/kafka/test/integration/admin_kafka_test.go

```diff
@@ -1171,6 +1171,7 @@ func TestAdminKafka_Update(t *testing.T) {
 				g.Expect(resp.StatusCode).To(gomega.Equal(http.StatusOK))
```

Comment:
can also add an integration test from the user facing perspective: there are some tests already in https://github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/blob/f9ab2b6af9c72e956df5e1833a3a2235a1b982aa/internal/kafka/test/integration/kafkas_test.go